From 2a1aedd1d9c1fbe3634a8534ae5bc7fccfd267e7 Mon Sep 17 00:00:00 2001
From: Alexei Cazacov <alexei.cazacov@qt.io>
Date: Fri, 26 Sep 2025 13:24:33 +0300
Subject: [PATCH 116/239] Doc: Add links to Qt Academy courses

Task-number: QTBUG-134130
Change-Id: Ib0844fa55f97492012d3911f5ac03d7ffc16a743
Reviewed-by: Jaishree Vyas <jaishree.vyas@qt.io>
(cherry picked from commit 3cda99d3ff2425f996e337967285e17c0d1859ba)
Reviewed-by: Qt Cherry-pick Bot <cherrypick_bot@qt-project.org>
---
 .../doc/src/getting-started/getting-started.qdoc   | 14 +++++++++++++-
 1 file changed, 13 insertions(+), 1 deletion(-)

diff --git a/src/quick/doc/src/getting-started/getting-started.qdoc b/src/quick/doc/src/getting-started/getting-started.qdoc
index 098b4436c4..6e6ac2c6f3 100644
--- a/src/quick/doc/src/getting-started/getting-started.qdoc
+++ b/src/quick/doc/src/getting-started/getting-started.qdoc
@@ -1,4 +1,4 @@
-// Copyright (C) 2021 The Qt Company Ltd.
+// Copyright (C) 2025 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR GFDL-1.3-no-invariants-only
 
 /*!
@@ -17,6 +17,18 @@ module, designers and developers can easily build fluid animated user interfaces
 in QML, and have the option of connecting these user interfaces to any back-end
 C++ libraries.
 
+\section1 Qt Academy Courses
+
+The following Qt Academy courses are recommended for getting started with QML
+and Qt Quick:
+
+\list
+\li \l {https://www.qt.io/academy/course-catalog#qml-for-beginners}{QML for Beginners}
+\li \l {https://www.qt.io/academy/course-catalog#introduction-to-qml}{Introduction to QML}
+\li \l {https://www.qt.io/academy/course-catalog#introduction-to-qt-quick}{Introduction to Qt Quick}
+\li \l {https://www.qt.io/academy/course-catalog#introduction-to-qt-quick-controls}{Introduction to Qt Quick Controls}
+\li \l {https://www.qt.io/academy/course-catalog#creating-a-simple-qt-quick-application-}{Creating a Simple Qt Quick Application}
+\endlist
 
 \section1 What is QML?
 
-- 
2.51.2

